puppet/modules/jenkins_job_builder/files/centos.org/pipelines/lib/foremanCentosJob.groovy

```diff
@@ -1,3 +1,5 @@
+playBook = pipelineVars(action: params.action, type: params.type, version: params.version, os: params.distro, extra_vars: ['foreman_expected_version': params.expected_version])
```

Comment:
Don't you need to remove `playbook = playBookVars()` on line 6 if you define it here?